agent/bench-scripts/pbench-run-benchmark.pl

```diff
@@ -394,17 +398,19 @@ sub signal_handler {
         close(BULK_FH);
         system(". ./bulk-sample.sh");
     } else {
-	system("pbench-end-tools --group=" . $tool_group . " --dir=" . $base_bench_dir);
+        system("pbench-end-tools --group=" . $tool_group . " --dir=" . $base_bench_dir);
```

Comment:
I could not find a reference so I figured it was best to add a definition at the top just in case, but this is likely a bug that has existed for a while.